CMakeLists.txt

```diff
@@ -65,6 +42,7 @@ endif()
 if(BUILD_OSX_UNIVERSAL)
 set(CMAKE_OSX_ARCHITECTURES "x86_64;arm64")
 endif(BUILD_OSX_UNIVERSAL)
+set(CMAKE_OSX_DEPLOYMENT_TARGET "10.9" CACHE STRING "Minimum OS X deployment version")
 
 #
 # Find the git hash if this is a working copy.
```

Comment:
We should probably use `git rev-parse --short` to get the commit hash. `git describe` gives me something like `v1.0.0-428-g160866fc` for current master.

However, we can do another PR for that particular change if preferred.

Reply:
@tmiw - would you like to try that change in this PR?  I'm happy to wait for you to try that...

Reply:
Done :)

Reply:
Thank you @tmiw :slightly_smiling_face: